test/java/org/opendatakit/briefcase/ui/export/ExportPanelPageObject.java

```diff
@@ -0,0 +1,51 @@
+package org.opendatakit.briefcase.ui.export;
+
+import static org.opendatakit.briefcase.ui.export.FormExportTableModel.SELECTED_CHECKBOX_COL;
+
+import javax.swing.JFrame;
+import org.assertj.swing.core.MouseButton;
+import org.assertj.swing.core.Robot;
+import org.assertj.swing.data.TableCell;
+import org.assertj.swing.edt.GuiActionRunner;
+import org.assertj.swing.fixture.FrameFixture;
+import org.assertj.swing.fixture.JButtonFixture;
+import org.opendatakit.briefcase.model.TerminationFuture;
+
+class ExportPanelPageObject {
+  private final ExportPanel exportPanel;
+  private FrameFixture window;
+
+  private ExportPanelPageObject(ExportPanel exportPanel, FrameFixture window) {
+    this.exportPanel = exportPanel;
+    this.window = window;
+  }
+
+  static ExportPanelPageObject setUp(Robot robot) {
```

Comment:
Nice correct capitalization of the phrasal verb `set up` (as opposed to the noun or adjective `setup`). We often don’t see that.